01-module-system/helpers.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs')
+const clr = require ("chalk")
+
+const getNotes = () => {
+    try{
+const data = fs.readFileSync('notes.json')
+const content = data.toString()
+const jsonData=JSON.parse(content)
+return jsonData 
+    }catch(err){
+        console.log(clr.red("ERROR! Invalid input format in 'notes.json'"))
+        return []
+    }
+}
+
+const saveNotes = (object) => {
+    const jsonObject = JSON.stringify(object)
+    fs.writeFileSync('notes.json',jsonObject)
+}
+
+const addNotes = (title, body) => { 
+    const jsonData = getNotes()
+    if(!jsonData.find(x=>{if(x.title===title) return true})){
+        jsonData.push({"title": title, "body": body})
+        saveNotes(jsonData)
+}
+else{
+    console.log('Title already exists')
+}
```

Comment:
fix indentation (tab), move else on the same line where if statement closes and leave space after else